driver.py

```diff
@@ -10,13 +10,11 @@
 # to connect to a ray cluster.
 # If RAY_ADDRESS begins with "anyscale://" then your program 
 # uses an Anyscale cluster to run Ray.
-os.environ["RAY_ADDRESS"]="anyscale://onboarding"
```

Comment:
Ah this is what @pcmoritz was referring to before.  Makes sense to remove it here.